### PR TITLE
fix: frequency when no quantity

### PR DESCRIPTION
**Pull request by TBonnin**
if frequency is set to `every hour` the connection page shows `h`.
This commit ensures such frequencies shows as `1h` (or `1d`, ...) 

Also adding abbreviation for month and year even though we don't have a lot of such syncs, it is just to be consistent

Screenshot 2024-07-02 at 10 38 11

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: packages/webapp/src/utils/utils.tsx [hunk 1]
```diff
@@ -218,10 +218,20 @@ export function formatFrequency(frequency: string): string {
         hours: 'h',
         hour: 'h',
         days: 'd',
-        day: 'd'
+        day: 'd',
+        months: 'mon',
+        month: 'mon',
```
Comment (bodinsamuel) [inline packages/webapp/src/utils/utils.tsx]: why not `M` less clear but consistent in term of size?

**Review by bodinsamuel — APPROVED** [review]

File: packages/webapp/src/utils/utils.tsx (hunk 1 above)
Comment (TBonnin) [reply packages/webapp/src/utils/utils.tsx]: I thought about that. As you said it is consistent but less clear. I don't mind either way. @khaliqgant any preference?

Comment (bodinsamuel) [reply packages/webapp/src/utils/utils.tsx]: not pushing just saying 👍🏻

Comment (khaliqgant) [reply packages/webapp/src/utils/utils.tsx]: month abbreviation should actually be `mo` if singular and `mos` if plural

**Review by khaliqgant — APPROVED** [review]

**Pull request merged**